scripts/release-em-squids.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Script for publishing the various ink! crates to `crates.io`.
+#
+# It is assumed that version numbers have already been bumped. It also doesn't do any Git
+# or Changelog related work.
+
+set -eux
+
+# This give us the "best" order in which to release our crates
+to_release=( $(cargo unleash to-release) )
+crates=()
```

Comment:
The variable is empty, hence the script can currently not be used.

In general: what's your idea how to continue with this PR?